dist/modules/jquery.fancytree.fixed.js

```diff
@@ -387,13 +387,12 @@
 
 				if (!$trRight && $trLeft.length) {
 					var idx = $trLeft.index(),
+						fixedWrapper = $(this.$fixedWrapper),
```

Comment:
`this.$fixedWrapper` is already a jQuery object, so no need to wrap inside `$()` again.
If you want to copy it into a local variable, it should be named `$fixedWrapper`, since `$` is our convention for jQuery object vars.
However, you may not need this var at all.

Reply:
Yes, but $fixedWrapper at the time of initialization it equals null, need to add logic, but I wanted to implement it easier